Example/Controllers/SwiftUI/Helper/MiniAppViewMessageDelegator.swift

```diff
@@ -183,6 +183,16 @@ class MiniAppViewMessageDelegator: NSObject, MiniAppMessageDelegate {
     func closeMiniApp(withConfirmation: Bool, completionHandler: @escaping (Result<Bool, MiniAppJavaScriptError>) -> Void) {
         onShoudCloseMiniApp?(withConfirmation)
     }
+
+    func getHostAppThemeColors(completionHandler: @escaping (Result<HostAppThemeColors?, MiniAppJavaScriptError>) -> Void) {
```

Comment:
`MiniAppJavaScriptError` is used to return any error that is validated by the SDK itself. You should be using MASDKError here.

Reply:
![Screenshot 2023-05-10 at 19 27 52](https://github.com/rakutentech/ios-miniapp/assets/115203844/c846646f-7d8b-4dd0-9e48-3978d2be890d)
If I use MASDKError I will get this.